### DISCUSSION TITLE
1.0.5pre6

### DISCUSSION THREAD

**Discussion by OStrama** [OStrama/weishaupt_modbus#79] [Announcements]
NL language included, thanks to https://github.com/StijnVdd !
When the powermap is interpolated, a png file containing the graph is tried to save in config_dir/www/local for display in bitmap card

<hr /><em>This discussion was created from the release <a href='https://github.com/OStrama/weishaupt_modbus/releases/tag/'>1.0.5pre6</a>.</em>

**Comment by maddinhome** [comment]
Hello, is there anything else going on here or have you stopped development? or are you on your well-deserved Christmas vacation?

**Reply by OStrama** [reply]
We do a little break and will continue in some days 😉

**Reply by maddinhome** [reply]
Thanks for the reply. 
I'm happy and looking forward to the future. 
Just by the way. I have attached my Powermap graphic file here. 
What does it say exactly?

![weishaupt_modbus_powermap](https://github.com/user-attachments/assets/29d20ce3-7943-4ba3-bb5c-d2080918cc39)

**Reply by OStrama** [reply]
It shows the power vs. temeprature curve that has been interpolated from the power map file. It should look equal to the documentation of your heatpump. You can add the image to an image contron in home assistant, too:
![image](https://github.com/user-attachments/assets/8c34dfd1-462a-480d-9c2a-c183d54698b9)
(However, I see that I have to adapt scaling ;-))

**Reply by maddinhome** [reply]
So you mean that's the heating curve? 
Like this:
![Heizkurve](https://github.com/user-attachments/assets/138e83c0-3365-4134-846f-9c8e65c9a2fa)

Or is it the curve of how much electricity I have to use to reach a certain temperature. But then I still don't understand the curve. Maybe it really is the scaling. So what you've already noticed.

Based on your graphics, I would have to use around 5900 watts at -20 C... I don't think I understand it. Sorry

**Reply by OStrama** [reply]
It should be basically this curve:
![image](https://github.com/user-attachments/assets/8973b0db-1d91-41ce-b461-5bcbf522709b)
It is the maximum heating power vs. outside temperature for a given feed temperature. Internally, we use this curve to calculate the heating power from the "Leistungsanforderung". Example: for my Biblock: For feed temp 35 °C at 2,5 °C outside temp the max heat power is 10 kW. If we have 35 % "Leistungsanforderung" the heat power is 3500 Watts

**Reply by maddinhome** [reply]
Ahh now... I can't find these types of graphics in my documentation yet. Thanks for the explanation. I only find the heating curve....
